datahub/wesim.py

```diff
@@ -0,0 +1,143 @@
+"""This module defines the data structures for the WESIM model."""
+
+import pandas as pd
+
+REGIONS_KEY = {
+    "Scotland": "SCO",
+    "North Engl&Wal": "NEW",
+    "Midlands": "MID",
+    "London": "LON",
+    "South Eng&Wal": "SEW",
+}
+
+INTERCONNECTORS_KEY = {"SCO-IE", "NEW-NOR", "NEW-IE", "SEW-CE"}
+
+
+def read_wesim() -> dict[int | str, pd.DataFrame]:
+    """Read the WESIM data from the excel file.
+
+    Returns:
+        pd.DataFrame: A Dictionary of DataFrames for each sheet in the file
+    """
+    excel = pd.read_excel(
+        "../1_Wesim_GB_hourly_data.xlsx",
```

Comment:
This is a good point. I was considering putting this file in the repo though. It's 178kb and it's very specifically structured, so it becomes very difficult to write tests without it. Thoughts? (although I think it's maybe a non-starter because that data probably shouldn't be public)